Makefile

```diff
@@ -6,23 +6,28 @@ SUBDIRS=utils libavpipe exc elvxc
 SRCS=avpipe_handler.c
 OBJS=$(SRCS:%.c=$(BINDIR)/%.o)
 
-.PHONY: all test clean
+.PHONY: all test clean help
 
 .DEFAULT_GOAL := dynamic
 
-all install: copy_libs check-env
+help: ## Show help
+	@grep -h -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+all install: copy_libs check-env ## Build C and Go artifacts
 	@for dir in $(SUBDIRS); do \
 	echo "Making $@ in $$dir..."; \
 	(cd $$dir; make $@) || exit 1; \
 	done
 
 dynamic: copy_libs_all all
 
-clean: lclean
+clean: lclean ## Clean C and Go build artifacts
 	@for dir in $(SUBDIRS); do \
 	echo "Making $@ in $$dir..."; \
 	(cd $$dir; make $@) || exit 1; \
 	done
+	@echo "Go clean"
```

Comment:
Not a good idea to clean go cache and everything in clean tag (it increase the build time),

It is better to add a new tag, like:

cleanall: clean goclean
  
goclean:
             @echo "Go clean"
             @go clean -modcache -testcache -cache -i -r

Reply:
Ok - I reverted the changes to Makefile since they are not necessary or related to this PR.  I can make the Go clean in a separate PR.